source/features/shorten-links.tsx

```diff
@@ -7,9 +7,13 @@ import features from '.';
 import {linkifiedURLClass} from '../github-helpers/dom-formatters';
 
 function init(): void {
-	observe(`a[href]:not(.${linkifiedURLClass})`, {
+	observe('a[href]', {
 		constructor: HTMLAnchorElement,
 		add(link) {
+			if (link.closest(linkifiedURLClass)) {
```

Comment:
```suggestion
			if (link.closest('code')) {
```

Right? We just don't want to shorten the links that appear in code. The only exception to this is review comments, so you also need to check whether the comment wrapper is "closer" than the code element.

For example the DOM:


```
code
  a -> don't shorten 
  review comment
    a -> shorten
    code suggestion in review comment
      a -> don't shorten